.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
+	"semi": false,
+	"singleQuote": true,
+	"printWidth": 180,
+	"useTabs": true
```

Comment:
I noticed files were formatted differently. I just based this off the Svelte file, but I don’t care at all what the format is.

Reply:
If you want to just revert the Prettier formatting that’s fine too! I just know myself (and other contributors) often have format on save turned on. And enforcing _some_ style that can be linted is helpful—whatever that is (again, couldn’t care less on what the formatting style is—your choice here)